scripts/start-setupVanillaTweaks

```diff
@@ -81,4 +83,20 @@ if [[ "$PACKS" ]] && [[ "$VT_VERSION" ]]; then
   fi
 fi
 
+# Download and move crafting tweaks
+if [[ "$CRAFTING_TWEAKS" ]] && [[ "$VT_VERSION" ]]; then
```

Comment:
After looking at tests, this needs to also be checking for JQ's null string. "null". :(
Something like
```bash
if [[ "$CRAFTING_TWEAKS" ]] && [[ "$CRAFTING_TWEAKS" != "null" ]] && [[ "$VT_VERSION" ]]; then
```

Reply:
I'll add this now

Reply:
Added it for the $PACKS variable too.